### ISSUE TITLE
RobotOS.jl for julia-1.0

### ISSUE THREAD

**Issue by gstavrinos** [jdlangs/RobotOS.jl#50]
Hey Josh,

Since julia is now on its first stable release, are you planning to update RobotOS.jl so it is compatible with it? I saw a pull request  (#48) that is for julia-0.7 (so also for julia-1.0) which could be a candidate.

Thanks,

George

**Comment by jdlangs** [comment]
Yes, I will get this package up to date within the next few weeks.

**Comment by gstavrinos** [comment]
Cool, thanks! You could maybe create some issues so we can help with the whole updating process.

**Comment by gstavrinos** [comment]
Thank you very much!
